docs/reference/search/search_context.asciidoc

```diff
@@ -1,15 +1,16 @@
 [[search-context]]
 ==== Search Context
 
-By default, a search request executes against the latest point in time readers of the
-target indices. Sometimes it's more preferred to execute multiple search requests using
-the same point in time readers. For example, the combined result of the initial and
-subsequent search_after requests would be more consistent if they use the same point
-in time readers. This can be done by using a single search context for all search requests.
-
-A search context must be opened in a separate step before being used in subsequent
-search requests. The keep_alive parameter tells Elasticsearch how long it should keep
-the seach context alive, e.g. `?keep_alive=5m`.
+A search request by default executes using the most recent searchers of the target indices.
```

Comment:
I really like this "point in time" notion that you had before. And also may be instead of "searchers" use "search context"?  May be a better phrasing could be:
 "A search request by default executes against the most recent point it time view of the target indices, which is called `search context`....

Reply:
++. I pushed https://github.com/elastic/elasticsearch/pull/56480/commits/ada43b3f9cd27033143d702d6a833a594bbc7e78